packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <div :class="[$style['c-restaurantCard-fees']]">
+        <icon-text
+            v-if="deliveryFeeText"
+            :class="[$style['c-restaurantCard-fees-item']]"
+            :text="deliveryFeeText"
+            :hide-icon-in-tile-view="true"
+            data-test-id="restaurant-fees-delivery">
+            <cash-small-icon data-test-id="restaurant-fees-delivery-icon" />
+        </icon-text>
+
+        <icon-text
+            v-if="minOrderText"
+            :class="[$style['c-restaurantCard-fees-item']]"
+            :text="minOrderText"
+            data-test-id="restaurant-fees-min-order" />
+    </div>
+</template>
+
+<script>
+import { CashSmallIcon } from '@justeat/f-vue-icons';
+import IconText from '../IconText.vue';
+
+export default {
+    name: 'RestaurantFees',
+    components: {
+        IconText,
+        CashSmallIcon
+    },
+    props: {
+        deliveryFeeText: {
+            type: String,
+            default: null
+        },
+        minOrderText: {
+            type: String,
+            default: null
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+.c-restaurantCard-fees {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.c-restaurantCard-fees-item {
+    position: relative;
+    padding-right: spacing();
+    margin-right: spacing(x0.5);
+    white-space: nowrap;
+    @include font-size($font-body-s-paragraph);
+
+    &:before {
+        display: none;
+    }
+
+    &:after {
+        content: '\2022'; // round bullet character
+        color: $color-content-subdued;
+        position: absolute;
+        right: 0;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+    }
+
+    &:last-of-type {
+        padding-right: 0;
```

Comment:
If you removed the absolute positioning from the `after` (maybe used lineheight and or translates to position on relative) and added the margin/padding to that you would'nt need to set and then remove padding/margin on the parent elm :)

Reply:
This is actually far more graceful, thanks for the suggestion! Just commiting the update now :)